src/adapters/leasing-adapter.ts

```diff
@@ -132,6 +132,23 @@ const getContactByContactCode = async (
   }
 }
 
+const getTenantByContactCode = async (
+  contactCode: string
+): Promise<AdapterResult<any, unknown>> => {
+  try {
+    const res = await axios.get(
+      `${tenantsLeasesServiceUrl}/tenant/contactCode/${contactCode}`
+    )
+
+    if (!res.data.data) return { ok: false, err: 'not-found' }
+
+    return { ok: true, data: res.data.data }
```

Comment:
Same remark as in the leasing PR, let's use the response structure in https://github.com/Bostads-AB-Mimer/onecore-core/commit/04f673d7f450ee396c416d948dc7c5ea6fe20a6a

Reply:
https://github.com/Bostads-AB-Mimer/onecore-core/pull/125/commits/d4edf8ca7d68f41eddd72fc5c31daef74786150b